bundles/CoreBundle/src/Migrations/Version20230107224432.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Pimcore\Model\Tool\SettingsStore;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230107224432 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Grant XLIFF import/export permission to those who already had "translation" permission, due XLIFF functionalities being moved out to be Standalone from Core Translation';
```

Comment:
```suggestion
        return 'Marks the XliffBundle as installed and grants XLIFF import/export permissions.';
```
![image](https://user-images.githubusercontent.com/5137917/212096372-97685661-3cb9-4784-a397-88659cdcd171.png)